### ISSUE TITLE
Battery Readings / Sleep inconsistency

### ISSUE THREAD

**Issue by astone1** [superzeldalink/Asus-Vivobook-S530-hackintosh#1]
The battery readout doesn't seem to update properly, also sometimes the system will randomly wake from sleep for no apparent reason.  MacOS 10.14.5

**Comment by superzeldalink** [comment]
1. The battery indicator won’t update when wake from sleep and the screen starts flickering too. I’m using iStatMenu for battery indicator now. I’m searching a way to fix this. 
2. Try using boot arg darkwake=0 to turn off PowerNap

**Comment by astone1** [comment]
Ok, will keep you posted.  Any advice on the battery readings?  Noticed in proprietary apps the reading seems to update properly.  It's only on the native status bar where it freezes.

**Comment by superzeldalink** [comment]
Some posts told setting abm_firstpolldelay=16000 or 32000 but it won’t work. Maybe this is SSDT issue

**Comment by astone1** [comment]
Yeah, I already have Darkwake=0.  Just realised this, must be related to the ssdt then.  Also, you tried those abm_firstpolldelay options and it didn't work?

**Comment by astone1** [comment]
abm_firstpolldelay=8000 fixed it I believe

**Comment by astone1** [comment]
Yeah, quite certain this fixed it!

**Comment by superzeldalink** [comment]
Let me try it

**Comment by astone1** [comment]
Yeah, so I put my system to sleep and than I ran something to increase CPU usage.  Battery seems to be updating properly now.

**Comment by superzeldalink** [comment]
It fixed :)
But are you facing the issue that screen starts flickering when charge if it wake from sleep ?

**Comment by astone1** [comment]
Yeah, I do occasionally get that haha.  It's very subtle though, right?  Also, it happens very randomly.

**Comment by astone1** [comment]
Also, do you ever get the issue where you sometimes randomly get really distorted audio out of the 3.5mm jack?

**Comment by superzeldalink** [comment]
Yes. But very rare, it’s only 1/10. So I don’t spend my time on that
